pkg/api/api.go

```diff
@@ -11,7 +11,7 @@ import (
 type API struct {
 	Kafka          func() kafkamgmtclient.DefaultApi
 	ServiceAccount func() kafkamgmtclient.SecurityApi
-	KafkaAdmin     func(kafkaID string) (kafkainstanceclient.DefaultApi, *kafkamgmtclient.KafkaRequest, error)
+	KafkaAdmin     func(kafkaID string) (kafkainstanceclient.TopicsApi, kafkainstanceclient.GroupsApi, *kafkamgmtclient.KafkaRequest, error)
```

Comment:
```suggestion
	KafkaAdmin     func(kafkaID string) (kafkainstanceclient, *kafkamgmtclient.KafkaRequest, error)
```

If we return the full API client like this, it can be used like: `_ = api.TopicsApi.GetTopic()`

Reply:
Yes. This requires refactor of the KeycloakConnection so it will take some time